### PR TITLE
front end skeleton

### PR DESCRIPTION
**Pull request by RyanHolmes**
💣

### CODE REVIEW

File: frontEnd/src/App.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class App extends Component {
+  constructor(props){
+    super(props)
+    this.state = {}
+  }
+
+  render() {
+    return (
+      <div>
+        <Landing></Landing>
```
Comment (hfiller) [inline frontEnd/src/App.js]: I think these can be written as `<Landing/>`

File: frontEnd/src/components/CreateUser.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+// import Bs, { Col } from 'react-bootstrap';
```
Comment (hfiller) [inline frontEnd/src/components/CreateUser.js]: remove

File: frontEnd/package 2.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "capstoneDemo",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "react-scripts": "0.8.4"
+  },
+  "dependencies": {
+    "react": "^15.4.2",
+    "react-bootstrap": "^0.30.7",
```
Comment (hfiller) [inline frontEnd/package 2.json]: 🔥

File: frontEnd/src/components/CreateUser.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    var scroll = Scroll.animateScroll;
+    scroll.scrollTo(1400);
+  }
+
+  render() {
+    return (
+      <div className="create-user-background">
+        <h1 className='page-title'> Create Users </h1>
+        <div className="create-user-container">
+          <UserCard img="../images/ryan.jpg" name="User 1"></UserCard>
```
Comment (hfiller) [inline frontEnd/src/components/CreateUser.js]: `<UserCard img="../images/ryan.jpg" name="User 1"/>`

File: frontEnd/src/components/EventCard.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import '../styles/EventCard.css';
+
+class EventCard extends Component {
+  render() {
+    return (
+      <div className="card event-card">
+        poop
```
Comment (hfiller) [inline frontEnd/src/components/EventCard.js:8]: this file?

File: frontEnd/src/components/Landing.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+var Scroll  = require('react-scroll');
+// import Bs, { Col } from 'react-bootstrap';
+
+import '../styles/Landing.css';
+import UserCard from './UserCard';
+
+class Landing extends Component {
+
+  scrollWin(){
```
Comment (hfiller) [inline frontEnd/src/components/Landing.js]: tabs on this file

File: frontEnd/src/components/Landing.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+var Scroll  = require('react-scroll');
+// import Bs, { Col } from 'react-bootstrap';
```
Comment (hfiller) [inline frontEnd/src/components/Landing.js]: 🔥

File: frontEnd/src/components/Landing.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    scroll.scrollTo(700);
+  }
+
+  render() {
+    return (
+      <div className="landing-background">
+        <h1 className="page-title white"> BlueShift </h1>
+        <div className="landing-container">
+          <h3 className="landing-description white">This site is a demo to showcase the recommendation engine we have created for Universe.com. It uses natural language processing to classify events and machine learning to do hybrid filtering. </h3>
+          <UserCard img="../images/ryan.jpg" name="Vanessa Deng"></UserCard>
```
Comment (hfiller) [inline frontEnd/src/components/Landing.js]: more instances to use the shorter format

File: frontEnd/src/components/RecommendEvents.js [hunk 9]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import Bs, { Col } from 'react-bootstrap';
```
Comment (hfiller) [inline frontEnd/src/components/RecommendEvents.js]: 🔥

File: frontEnd/src/components/UserCard.js [hunk 10]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import '../styles/UserCard.css';
+
+class UserCard extends Component {
+  render() {
+    // var img = require(`${this.props.img}`);
```
Comment (hfiller) [inline frontEnd/src/components/UserCard.js:6]: what is this?

File: frontEnd/src/styles/CreateUser.css [hunk 11]
```diff
@@ -0,0 +1,13 @@
+.create-user-container {
+  margin-left: 200px;
+  width: 80%;
+  height: 100%;
+}
+
+.create-user-background {
+  /*TODO - margins messed up*/
```
Comment (hfiller) [inline frontEnd/src/styles/CreateUser.css]: 💣

**Review by hfiller — DISMISSED** [review]
Looks great! just small changes

File: frontEnd/src/App.js (hunk 1 above)
Comment (RyanHolmes) [reply frontEnd/src/App.js]: ✅

File: frontEnd/src/components/CreateUser.js (hunk 2 above)
Comment (RyanHolmes) [reply frontEnd/src/components/CreateUser.js]: ✅

File: frontEnd/package 2.json (hunk 3 above)
Comment (RyanHolmes) [reply frontEnd/package 2.json]: ✅

File: frontEnd/src/components/CreateUser.js (hunk 4 above)
Comment (RyanHolmes) [reply frontEnd/src/components/CreateUser.js]: ✅

File: frontEnd/src/components/EventCard.js (hunk 5 above)
Comment (RyanHolmes) [reply frontEnd/src/components/EventCard.js:8]: needed for later

File: frontEnd/src/components/Landing.js (hunk 7 above)
Comment (RyanHolmes) [reply frontEnd/src/components/Landing.js]: 🐳

File: frontEnd/src/components/Landing.js (hunk 6 above)
Comment (RyanHolmes) [inline frontEnd/src/components/Landing.js:9]: merp merp

File: frontEnd/src/components/Landing.js (hunk 8 above)
Comment (RyanHolmes) [reply frontEnd/src/components/Landing.js]: ✅

File: frontEnd/src/components/RecommendEvents.js (hunk 9 above)
Comment (RyanHolmes) [reply frontEnd/src/components/RecommendEvents.js]: ✅

File: frontEnd/src/components/UserCard.js (hunk 10 above)
Comment (RyanHolmes) [reply frontEnd/src/components/UserCard.js:6]: for later

File: frontEnd/src/styles/CreateUser.css (hunk 11 above)
Comment (RyanHolmes) [reply frontEnd/src/styles/CreateUser.css]: ✅

**Comment by iancawood** [comment]
Down the road we should add bootstrap for responsiveness and we need to make sure that the `scroller.scrollTo();` lines are the correct amount, but as a skeleton it's good.

**Comment by iancawood** [comment]
As a recommendation though, define a global default fault so that the fakt class doesn't need to be added to everything. 

We could make a style guide, but that could be overkill.

**Comment by RyanHolmes** [comment]
:green_salad:

**Comment by hfiller** [comment]
:shipit:

**Pull request merged**